lara-typescript/src/example-interactive/src/components/runtime.tsx

```diff
@@ -22,6 +22,11 @@ export const RuntimeComponent: React.FC<Props> = ({initMessage}) => {
     }
   }, [authoredState]);
 
+  const [customMessages, setCustomMessages] = useState<ICustomMessage[]>([]);
+  useCustomMessages((msg: ICustomMessage) => {
```

Comment:
It would be easier for developers (I think), if this hook also took an argument of the custom messages supported by the interactive. Then the hook can take care of adding those to the supportedFeatures.  That way the developer doesn't need to worry about the details of how those supported custom messages are published via the api.

However this automated approach might complicate how setSupportedFeatures is used. For example in the question-interactives repo here: https://github.com/concord-consortium/question-interactives/blob/6c0453518d15ea30e8e88a1ea84c1fa9d6a13485/src/image/components/runtime.tsx#L37

Reply:
Done